src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private String line;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        int result = 0;
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String type = parts[0];
+                int value = Integer.parseInt(parts[1]);
+                if (type.equals("supply")) {
+                    supply += value;
+                } else if (type.equals("buy")) {
```

Comment:
let's create constants for all operation names, ,magic numbers and coma separator